{
  assertTerminates();
  assertNull("No error should have been thrown, but " + getName() + " completed error",error);
}
